csharp/sources/Valkey.Glide/BaseClient.HashCommands.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+using Valkey.Glide.Commands;
+using Valkey.Glide.Internals;
+
+namespace Valkey.Glide;
+
+public abstract partial class BaseClient : IHashCommands
+{
+    public async Task<ValkeyValue> HashGetAsync(ValkeyKey key, ValkeyValue hashField, CommandFlags ignored = CommandFlags.None)
+        => await Command(Request.HashGetAsync(key, hashField));
```

Comment:
And we need to check whether flags are not set. See `Utils.Requires`

Reply:
We need to create a new PR since this should be applied to all commands. Currently we don't have any parameter guard clauses on any command.